packages/netlify-cms-core/src/components/Workflow/WorkflowCard.js

```diff
@@ -21,7 +21,6 @@ const styles = {
 const WorkflowLink = styled(Link)`
   display: block;
   padding: 0 18px 18px;
-  padding: 0 18px 18px;
```

Comment:
Duplicate line?

Reply:
in my defense I was very sleep deprived lol